src/scss/_Header.scss

```diff
@@ -9,6 +9,7 @@ header {
 	font-family: $title-font;
 	font-size: 1.75em;
 	color: $rancid-green;
+	box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.562);
```

Comment:
You're right. The shadow here does give his header more depth. +1

Reply:
sweeet. It's all about the details.